can/io/asc.py

```diff
@@ -139,18 +150,18 @@ def on_message_received(self, msg):
         else:
             dtype = "d {}".format(msg.dlc)
             data = ["{:02X}".format(byte) for byte in msg.data]
+
         arb_id = "{:X}".format(msg.arbitration_id)
-        if msg.id_type:
-            arb_id = arb_id + "x"
-        timestamp = msg.timestamp
-        if timestamp >= self.started:
-            timestamp -= self.started
+        if msg.is_extended_id:
+            arb_id += "x"
 
         channel = msg.channel if isinstance(msg.channel, int) else self.channel
-        line = self.LOG_STRING.format(time=timestamp,
+
+        line = self.LOG_STRING.format(time=msg.timestamp,
```

Comment:
[Here](https://bitbucket.org/tobylorenz/vector_asc/src/47556e1a6d32c859224ca62d075e1efcc67fa690/src/Vector/ASC/tests/unittests/data/CAN_Log_Trigger_3_2.asc?at=master&fileviewer=file-view-default) is an example of an ASC file. There is not much info on this format but my interpretation is that `Begin Triggerblock Mon Mar 7 01:21:51 pm 2005` defines the start of measurement and the timestamps are relative to this time.